addons/website/tests/test_configurator.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from unittest.mock import patch
+
+import odoo.tests
+from odoo.addons.iap.tools.iap_tools import iap_jsonrpc_mocked
+from odoo.tools import mute_logger
+
+class TestConfiguratorCommon(odoo.tests.HttpCase):
+
+    def _theme_upgrade_upstream(self):
+        # patch to prevent module install/upgrade during tests
+        pass
+
+    def setUp(self):
+        super().setUp()
+
+        def iap_jsonrpc_mocked_configurator(*args, **kwargs):
+            endpoint = args[0]
+            params = kwargs.get('params', {})
+            language = params.get('lang', 'en_US')
+            if endpoint.endswith('/api/website/1/configurator/industries'):
+                if language == 'fr_FR':
+                    return {"industries": [
+                        {"id": 1, "label": "abbey in fr"},
+                        {"id": 2, "label": "aboriginal and torres strait islander organisation in fr"},
+                        {"id": 3, "label": "aboriginal art gallery in fr"},
+                        {"id": 4, "label": "abortion clinic in fr"},
+                        {"id": 5, "label": "abrasives supplier in fr"},
+                        {"id": 6, "label": "abundant life church in fr"}]}
+                else:
+                    return {"industries": [
+                        {"id": 1, "label": "abbey"},
+                        {"id": 2, "label": "aboriginal and torres strait islander organisation"},
+                        {"id": 3, "label": "aboriginal art gallery"},
+                        {"id": 4, "label": "abortion clinic"},
+                        {"id": 5, "label": "abrasives supplier"},
+                        {"id": 6, "label": "abundant life church"}]}
+            elif '/api/website/2/configurator/recommended_themes' in endpoint:
+                return []
+            elif '/api/website/2/configurator/custom_resources/' in endpoint:
+                return {'images': {}}
+
+            iap_jsonrpc_mocked()
+
+        iap_patch = patch('odoo.addons.iap.tools.iap_tools.iap_jsonrpc', iap_jsonrpc_mocked_configurator)
+        iap_patch.start()
+        self.addCleanup(iap_patch.stop)
+
+        patcher = patch('odoo.addons.website.models.ir_module_module.IrModuleModule._theme_upgrade_upstream', wraps=self._theme_upgrade_upstream)
+        patcher.start()
+        self.addCleanup(patcher.stop)
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestConfiguratorTranslation(TestConfiguratorCommon):
+
+    def test_01_configurator_translation(self):
+        with mute_logger('odoo.addons.base.models.ir_translation'):
```

Comment:
Do we really need this? Why is lang_install raising error?

Reply:
Ok I see, I don't like that:
- this might be muting useful error one day if I am not mistaken
- this is just to clean a bit the logs, which are.. logs

At that point, if that's just for cleaning, we might as well hide the remaining log entries about .po, as this is really not relevant in this PR since we create a new language and there is not a single .po file for it obviously. That step is ultra fast since no .po, might as well get rid of that log.

Anyway, that's not a big deal, you can keep it like that, the code where you found this was done by @mart-e so I trust that code ;-)

Reply:
I already did things like this (e.g. [`test_export_import`](https://github.com/odoo/odoo/blob/336c9aab3619abb5bfbeb07f87ae014facdefb0b/odoo/addons/test_translation_import/tests/test_term_count.py#L189-L190)). I think it was indeed to avoid too much noise. There is a redundant log in `trans_load` and `_load_module_terms`. Now that the translation system is a bit more mature, we could probably get rid (or lower to debug) one of them.

Reply:
Thanks for the explanation, that's then indeed to reduce "noise". So @detrouxdev you can keep it like that for this new test in stable :+1: